### DISCUSSION TITLE
Howto Connect Floccus?

### DISCUSSION THREAD

**Discussion by getwet** [floccusaddon/floccus#1494] [Q&A]
I had an error E017 on my Android App this morning, so I tried a reconnect. First I tried the App Token way to my nextcloud. After add my token the App told me, it is connected and I can close the window, but witch window? The only I can close is the small X right side from the address field. But if I close this windows I'm back in the dialog to add my nextcloud url. If I close the App, there's no account configured. May I do something wrong?

Best getwet

**Comment by marcelklehr** [comment]
Hi @getwet 
Please don't open issues in the Q&A discussions, we use the issue tab for this.
